docs/samples/Microsoft.ML.Samples/Dynamic/PermutationFeatureImportance.cs

```diff
@@ -80,49 +81,47 @@ public static void PFI_Regression()
 
             // Get the feature indices sorted by their impact on R-Squared
             var sortedIndices = permutationMetrics.Select((metrics, index) => new { index, metrics.RSquared })
-                .OrderByDescending(feature => Math.Abs(feature.RSquared))
+                .OrderByDescending(feature => Math.Abs(feature.RSquared.Mean))
                 .Select(feature => feature.index);
 
             // Print out the permutation results, with the model weights, in order of their impact:
-            // Expected console output:
-            //    Feature            Model Weight    Change in R - Squared
-            //    RoomsPerDwelling      50.80 -0.3695
-            //    EmploymentDistance   -17.79 -0.2238
-            //    TeacherRatio         -19.83 -0.1228
-            //    TaxRate              -8.60  -0.1042
-            //    NitricOxides         -15.95 -0.1025
-            //    HighwayDistance        5.37 -0.09345
-            //    CrimesPerCapita      -15.05 -0.05797
-            //    PercentPre40s         -4.64 -0.0385
-            //    PercentResidental      3.98 -0.02184
-            //    CharlesRiver           3.38 -0.01487
-            //    PercentNonRetail      -1.94 -0.007231
+            // Expected console output for 100 permutations:
+            //    Feature             Model Weight    Change in R-Squared    Standard Error of the Mean Change in R-Squared
```

Comment:
Would a direct 0.95 Confidence Interval be more useful to (the normal) user than Standard Error? Granted this pushes them into a specific CI, but I'm worried they will directly graph the StdErr, see non-overlapping error bars and make unwarranted decisions. 

Also, could present as: `-0.42940 ± 0.00637`

I see your Standard Error sometimes reports 7-digits past the decimal point. Maybe (5) would be useful? The counter case would be extremely tight error bars of ± 0.0000005. I'm doubtful we ever know (or is useful to know) our metrics to 7-decimal places. Generally, (4) digits is more than sufficient. Another route is always keeping ~two significant digits?

Reply:
For the example, I switched to print out a 95% Confidence confidence interval.

For whether we provide a confidence interval, I think it's enough to provide the basic statistical properties, and let the user derive the proper confidence interval -- they may want a 99% confidence interval, for example, and we don't want to have to export every possible value.

For significant digits, I think that this is also something that we should let the user determine. We are essentially providing a length (e.g. estimate of standard deviation), so truncating or rounding doesn't make sense -- better to give our best estimate, which is the full float we calculated.